import/faker/src/faker.utils.ts

```
@@ -0,0 +1,185 @@
+}
+
+function generateEnumData(config: any): string {
+  if (config && config.options) {
+    const options = config.options as string[]
+    return faker.helpers.arrayElement(options)
+  }
+  throw new Error('Enum field is missing options')
+}
+
+function generateReferenceData(config: any): string {
+  if (config && config.ref) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)